software/util/reorg.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""A Tool to help maintain the set of ttl files in schema.org.
+
+This helps reformatting, merging and splitting ttl files along
+semantic axis.
+"""
+
+from collections.abc import Sequence
+from rdflib import Graph, Namespace
+import logging
+import sys
+import argparse
+
+def SchemaOrgGraph():
```

Comment:
Consider adding type annotations as this is a new file, i.e. 

def SchemaOrgGraph() -> rdflib.Graph: